### PR TITLE
Add encryption keys to the PTech

### PR DESCRIPTION
**Pull request by crazybrain23**
## About the PR

This will add encryption keys to the HoP's PTech vending machine so that they can grant crew access to other radios, either when someone changes their job, or if a head requests it.

**Media**

image
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: crazybrain
- tweak: Added various encryption keys to the PTech.

### CODE REVIEW

**Comment by deltanedas** [comment]
this seems really nice if you break in to hops place

**Comment by EmoGarbage404** [comment]
i mean, you have access to possible AA if you break into hops.

File: Resources/Prototypes/Catalog/Cargo/cargo_vending.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockPTechFilled
-  cost: 800
+  cost: 1200
```
Comment (AlexMorgan3817) [inline Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:95]: Seems like not related to PR, what is this?

**Review by AlexMorgan3817 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Cargo/cargo_vending.yml (hunk 1 above)
Comment (crazybrain23) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:95]: It's the price for the restock crate. If it's not set higher than the value of the vending machine contents, then cargo gets an infinite money glitch and it will fail tests.

**Review by mirrorcult — APPROVED** [review]

File: Resources/Prototypes/Catalog/Cargo/cargo_vending.yml (hunk 1 above)
Comment (AlexMorgan3817) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:95]: Why it is in PR that not related to cargo...? you didn't described this change, this why i asking.
"Add encryption keys to the PTech" is not "Fixing infinite money glitch in cargo"

Comment (crazybrain23) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:95]: Because you need to change the price of the restock boxes when you change the contents of a vending machine.
```
# If you've arrived here after a failed NoCargoOrderArbitrage test, it's likely
# because the product in question contains a restock box which links to a
# vending machine inventory that has had items added to it recently and caused
# its calculated price to exceed the cost below.
#
# The costs will need to be kept in line with the inventory of the respective
# vending machines. An extra hundred or two profit margin should be fine.
````

Comment (AlexMorgan3817) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:95]: Aaaaa, wawawa, I just understood that this is a package for restock le encryption key vendor, sorry
